Cargo.toml

```diff
@@ -16,29 +16,29 @@ codegen-units = 1
 panic = "abort"
 
 [dependencies]
-anyhow = "1.0.32"
-async-process = "0.1.1"
-async-std = { version="1.6.3", features=["attributes", "unstable"] }
-cargo_metadata = "0.11.3"
-console = "0.13.0"
-dunce = "1.0.1"
-envy = "0.4.1"
-fs_extra = "1.2.0"
-futures = "0.3.5"
-http-types = "2.4.0"
-indicatif = "0.15.0"
-nipper = "0.1.8"
-notify = "4.0.15"
-open = "1.4.0"
+anyhow = "1"
+async-process = "1"
+async-std = { version="1.9", features=["attributes", "unstable"] }
```

Comment:
Would we be able to switch to `tokio` for `0.8`? It would simplify a lot of things that way.

Reply:
I'm planning on finishing up the review on the Tokio PR & the wasm-opt PR for the 0.9.0 release. Mostly just because the Tokio PR constitutes a fairly large number of changes, and I want to take a little more time to review and test it. So for now, just updating deps before we cut over to Tokio and such.

Reply:
Sure thing!